service-statistics-common/src/main/java/fi/nls/oskari/control/statistics/data/StatisticalIndicatorDataDimension.java

```diff
@@ -51,7 +51,13 @@ public void addAllowedValue(String value, String label) {
         addAllowedValue(new IdNamePair(value, label));
     }
     public void addAllowedValue(IdNamePair pair) {
-        allowedValues.add(pair);
+        // the values need to be sorted so doing "set-like" functionality here
+        if(!getAllowedValues().stream()
```

Comment:
For now it doesn't need to. For possible optimization I guess it might be a nice touch. But as the existing methods don't have a return value I'll just leave the overloaded one behaving the same way.